tracing/tracing.hh

```diff
@@ -674,7 +676,7 @@ void one_session_records::data_consumed() {
     }
 
     _is_pending_for_write = false;
-    budget_ptr = tracing::get_local_tracing_instance().get_cached_records_ptr();
+    budget_ptr = _local_tracing_ptr->get_cached_records_ptr();
```

Comment:
The same here -- the caller of data_consumed() is tracing_keyspace_helper which has the tracing& _local_tracing reference inherited from i_tracing_backend_helper.